bot/code_review_bot/__init__.py

```diff
@@ -100,12 +125,32 @@ def as_error(self):
         """
         raise NotImplementedError
 
-    @abc.abstractmethod
     def as_dict(self):
         """
         Build the serializable dict representation of the issue
         Used by debugging tools
         """
+        return {
+            "analyzer": self.analyzer,
+            "path": self.path,
+            "line": self.line,
+            "nb_lines": self.nb_lines,
+            "column": self.column,
+            "check": self.check,
+            "level": self.level,
+            "message": self.message,
+            "in_patch": self.revision.contains(self),
+            "is_new": self.is_new,
+            "validates": self.validates(),
+            "publishable": self.is_publishable(),
+            "extras": self.build_extra_informations(),
+        }
+
+    def build_extra_informations(self):
```

Comment:
```suggestion
    def build_extra_information(self):
```

Reply:
I'll merge extra infos & extra identifiers

Reply:
I removed them, as they are not encessary to build the frontend nor the hash (i left the extra identifiers workflow without any implementation just in case)